apps/autoscheduler/Featurization.h

```diff
@@ -13,6 +13,11 @@ struct PipelineFeatures {
         std::memset(this, 0, sizeof(PipelineFeatures));
     }
 
+    static constexpr int NumFeatures() {
```

Comment:
nit: We're doing underscores for methods and variables: num_features

Reply:
Fixed